great-idea/index.html

```diff
@@ -17,7 +17,63 @@
   <body>
```

Comment:
The title (inside the head part of your index.html page) should be Great Idea! - Services